docs/appendices/migration-guides/index.mdx

```diff
@@ -14,6 +14,69 @@ import TabItem from "@theme/TabItem";
 
 ## `2.17.0` -> `2.18.0`
 
+### The Prismic module has moved to a new home.
+
+We have tagged the last version of the Prismic module in the
+[external repository](https://gitlab.com/front-commerce/front-commerce-prismic)
+`v1` 🥳
+
+Since this version we have moved the Prismic module
+[into the core repository](https://gitlab.com/front-commerce/front-commerce/-/tree/main/modules/prismic)
+of Front-Commerce, moving forward any new features will be made available and
+follow the same
+[semantic versioning](http://localhost:3002/docs/appendices/release-process#semantic-versioning)
+as the core.
+
+You can use the module by adding the following changes to your
+`.front-commerce.js` config file:
+
+```diff title=".front-commerce.js"
+module.exports = {
+  modules: [
+    "./node_modules/front-commerce/theme-chocolatine",
+-   "./node_modules/front-commerce/modules/prismic",
++   "./node_modules/front-commerce-prismic/prismic",
```

Comment:
Shouldn't it be like this? I thought the front-commerce-prismic was now deprecated 🤔 If not I think there is some confusion in the description above.
```suggestion
-   "./node_modules/front-commerce-prismic/prismic",
+   "./node_modules/front-commerce/modules/prismic",
```

Reply:
I switched them when creating the diff 🤦 thanks